src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml

```diff
@@ -8,6 +8,10 @@
              xmlns:designTime="clr-namespace:Microsoft.R.Components.ConnectionManager.Implementation.View.DesignTime"
              xmlns:viewModel="clr-namespace:Microsoft.R.Components.ConnectionManager.ViewModel"
              xmlns:controls="clr-namespace:Microsoft.R.Wpf.Controls;assembly=Microsoft.R.Wpf"
+             xmlns:imaging="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.Imaging"
```

Comment:
These are no shell dependent. Components already reference Text.* and CoreUtility. Same layer as R.Editor.